package.json

```diff
@@ -21,7 +21,9 @@
     "serve": "yarn build && cd ./public && ws",
     "develop": "node . develop",
     "test": "export BROWSERSLIST_IGNORE_OLD_DATA=true && nyc mocha",
-    "test:tdd": "yarn test --watch"
+    "test:tdd": "yarn test --watch",
+    "lint:css": "stylelint './www/**/*.js', './www/**/*.css'",
```

Comment:
thanks @aholtzman !

A couple tweaks here I think:
1. Let's just have `lint:css`
1. Then, to use w/ `--fix`, then we can just do `yarn lint:css --fix`, since Yarn will forward any flags
    - similar to how `yarn test:tdd` builds off `yarn test`
1. Lastly, let's include `lint:css` to the main `lint` task

Bonus points:
1. Create `lint:js` script and move the eslint commands there

----

So basically all said and done, we would have something like this:
```json
"scripts": {
   
   "lint": "ls-lint && lint:js && lint:css",
   "lint:js": "eslint \"*.js\" \"./packages/**/**/*.js\" \"./test/*.js\" \"./www/**/**/*.js\"",
   "lint:css": "stylelint \"./www/**/*.js\", \"./www/**/*.css\"
   
}
```

This may trigger a build failure and require fixing of our source code, which is fine by me.